blink.js

```diff
@@ -1 +1,12 @@
 // YOUR CODE GOES HERE
+
+$.fn.blink = function( msecs ){
+  var that = $(this);
+  setInterval(
+    function()
+      {
+        if (that.css("color") == 'rgb(0, 0, 0)') { that.css("color", "#ffffff"); }
+        else { that.css("color", "#000000"); }
```

Comment:
For readability's sake, I'd move each of these statements to it's own line.

https://twitter.com/mixonic/status/656935651586256896

Also, this function isn't long so it's not a _huge_ deal, but you might consider breaking this into named methods so it reads even more clearly: `if visible, hide, else show`.